Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.11
+
+COPY docs /docs
+COPY entry.sh /
+RUN chmod +x /entry.sh
```

Comment:
I think you can remove this line if you change the permissions of the committed file

Reply:
But if someone commits from windows ? 🤔

Reply:
They can still set the permissions. Also, git should preserve the permissions if they are already set
https://stackoverflow.com/a/13593391

Reply:
Nice! Didn't know that :)

Reply:
✅